### PR TITLE
Feat/i18n

### PR DESCRIPTION
**Pull request by yuluyi**

### CODE REVIEW

File: src/lib/constant/errno.ts [hunk 1]
```diff
@@ -8,27 +9,27 @@ const errno = {
   metaMaskUserDeniedMessageSignature: 4001,
   metaMaskWalletRequestPermissions: -32002,
   metaMaskUserRejectedAccountAccess: -32603,
-  metaMaskReplacementTransactionUnderpriced: 'replacement transaction underpriced',
-  metaMaskTransactionHasBeenAborted: 'the transaction has been aborted',
+  metaMaskReplacementTransactionUnderpriced: t`replacement transaction underpriced`,
```
Comment (web3max) [inline src/lib/constant/errno.ts:12]: These texts are used as error codes and do not need to be translated.

File: src/lib/store/index.ts [hunk 2]
```diff
@@ -46,6 +47,7 @@ export interface ICKBAddressItem {
   notes?: string
 }
 
+// eslint-disable-next-line lingui/no-unlocalized-strings
```
Comment (web3max) [inline src/lib/store/index.ts:50]: Can this eslint rule just be disabled?

File: src/locales/en.json [hunk 3]
```diff
@@ -0,0 +1,94 @@
+{
```
Comment (web3max) [inline src/locales/en.json:1]: The i18n multi-language json file is not useful and should be deleted.

File: public/locales/en.js [hunk 4]
```diff
@@ -0,0 +1 @@
...camera permission.\",\"TIC2wW\":\"Secure your assets by adding extra trusted devices like mobile phones, computers or tablets immediately.\",\"a1SmQh\":\"Select Wallet\",\"M7wPsD\":\"Switch\",\"ADQ2w8\":\"Switch Address\",\"emwlZI\":\"The contract verification failed.\",\"mncaot\":\"The current wallet environment does not support payments using TRX, please upgrade your wallet version or register with another wallet.\",\"au4g2p\":\"The new trusted device has full control over this address.\",\"l5QXS/\":\"the transaction has been aborted\",\"7wVond\":\"The transaction has been aborted.\",\"ohV77q\":\"This device or browser unsupported. Try connect by social or wallet.\",\"x4f8pZ\":\"Tips\",\"QGs22D\":\"transactionBuilder.addUpdateData\",\"VDX7Kz\":\"Trusted Devices of CKB Address\",\"LobdAW\":\"Unknown Error\",\"7P0626\":\"Unknown error happened. Please try again or paste the data directly.\",\"SoaXFN\":\"Use the QR code or link below on your other trusted devices and follow the instructions inside.\",\"OnLxXh\":\"User cancelled login\",\"B/NS7w\":\"user closed popup\",\"Hrrv+f\":\"User rejected the transaction\",\"jvvy/0\":\"user_canceled\",\"uny84M\":\"ValidationFailure(-31)\"}")};
```
Comment (web3max) [inline public/locales/en.js:1]: This directory and the following multi-language files are not useful to be deleted.

**Review by web3max — APPROVED** [review]

File: src/lib/constant/errno.ts (hunk 1 above)
Comment (yuluyi) [reply src/lib/constant/errno.ts:12]: Some of the errors can be translated to human readable messages, therefore is included in i18n. Though the actual message is left untranslated in the po file, we can add translation in the future to enhance user experience.

File: src/lib/store/index.ts (hunk 2 above)
Comment (yuluyi) [reply src/lib/store/index.ts:50]: The rule is added explicitly. So we won't miss a translation by accident.

File: public/locales/en.js (hunk 4 above)
Comment (yuluyi) [reply public/locales/en.js]: Done

File: src/locales/en.json (hunk 3 above)
Comment (yuluyi) [reply src/locales/en.json]: Done

**Pull request merged**